test/client/fixtures/utils/html-test.js

```diff
@@ -31,14 +31,24 @@ test('hover marker', function () {
                '<div ' + INTERNAL_ATTRS.hoverPseudoClass + '></div>' +
                '<div ' + INTERNAL_ATTRS.hoverPseudoClass + '=""></div>';
 
-    var expexted     = '<a href="http://domain.com"></a><div></div><div></div>';
-    var pocessedHtml = htmlUtils.processHtml(html);
+    var expected      = '<a href="http://domain.com"></a><div></div><div></div>';
+    var processedHtml = htmlUtils.processHtml(html);
 
-    strictEqual(htmlUtils.cleanUpHtml(pocessedHtml), expexted);
+    strictEqual(htmlUtils.cleanUpHtml(processedHtml), expected);
 
     urlUtils.getProxyUrl = storedGetProxyUrl;
 });
 
+test('focus marker and autocomplete stored', function () {
```

Comment:
I don't see test code for autocomplete attribute.

I propose to have to two tests:
- `focus marker`
 - `autocomplete attribute`

Note, we have the special logic in `hasAttribute` for `autocomplete` attribute. See https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/fixtures/sandbox/node/attributes-test.js#L583